lerobot/scripts/push_dataset_to_hub.py

```diff
@@ -147,39 +128,47 @@ def push_videos_to_hub(repo_id: str, videos_dir: str | Path, revision: str | Non
 
 
 def push_dataset_to_hub(
-    data_dir: Path,
-    dataset_id: str,
-    raw_format: str | None,
-    community_id: str,
-    revision: str,
-    dry_run: bool,
-    save_to_disk: bool,
-    tests_data_dir: Path,
-    save_tests_to_disk: bool,
-    fps: int | None,
-    video: bool,
-    batch_size: int,
-    num_workers: int,
-    debug: bool,
+    raw_dir: Path,
+    raw_format: str,
+    repo_id: str,
+    push_to_hub: bool = True,
+    local_dir: Path | None = None,
+    fps: int | None = None,
+    video: bool = True,
+    batch_size: int = 32,
+    num_workers: int = 8,
+    episodes: list[int] | None = None,
+    force_override: bool = False,
+    cache_dir: Path = Path("/tmp"),
+    tests_data_dir: Path | None = None,
 ):
-    repo_id = f"{community_id}/{dataset_id}"
-
-    raw_dir = data_dir / f"{dataset_id}_raw"
-
-    out_dir = data_dir / repo_id
-    meta_data_dir = out_dir / "meta_data"
-    videos_dir = out_dir / "videos"
-
-    tests_out_dir = tests_data_dir / repo_id
-    tests_meta_data_dir = tests_out_dir / "meta_data"
-    tests_videos_dir = tests_out_dir / "videos"
-
-    if out_dir.exists():
-        shutil.rmtree(out_dir)
-
-    if tests_out_dir.exists() and save_tests_to_disk:
-        shutil.rmtree(tests_out_dir)
+    # Check repo_id is well formated
+    if len(repo_id.split("/")) != 2:
+        raise ValueError(
+            f"`repo_id` is expected to contain a community or user id `/` the name of the dataset (e.g. 'lerobot/pusht'), but contains '{repo_id}'."
+        )
 
+    # Robustify when `local_dir` is str instead of Path
+    if local_dir is not None:
+        local_dir = Path(local_dir)
+
+    # Send warning if local_dir isn't well formated
+    if local_dir is not None:
+        user_id, dataset_id = repo_id.split("/")
```

Comment:
Unless I'm missing something here, this line shouldn't be under `if local_dir is not None:` because `local_dir` is optional and `dataset_id` is used further down outside of this condition.

Reply:
Moved at the top
```python
# Check repo_id is well formated
if len(repo_id.split("/")) != 2:
    raise ValueError(
        f"`repo_id` is expected to contain a community or user id `/` the name of the dataset (e.g. 'lerobot/pusht'), but contains '{repo_id}'."
    )
user_id, dataset_id = repo_id.split("/")
```